package/content/example.php

```diff
@@ -0,0 +1,140 @@
+<?php
+$requestCount = $i ?? 0;
+$requestTime = $duration ?? 0;
+?>
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```

Comment:
Can you please remove this file for now? (or at least switch to HTML5)?

Reply:
I copied this from `dnf install caddy`'s index.html file.